music21/spanner.py

```diff
@@ -429,14 +429,16 @@ def addSpannedElements(self,
         True
         '''
         # presently, this does not look for redundancies
-        if not common.isListLike(spannedElements):
-            spannedElements = [spannedElements]
+        if isinstance(spannedElements, base.Music21Object):
+            spannedEls = [spannedElements]
+        else:
+            spannedEls = spannedElements
```

Comment:
If this feels too homonymous, I can just add a disable?

Reply:
I'd prefer the disable here -- there's no difference between spannedElements and spannedEls and it's just more confusing to me.  I don't consider rewriting an argument to be a problem.  And we'll have a lot more of this when we get to Stream etc.

I think I wrote "List" where Sequence is fine.